File: 1-Iniciante/ex1075.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let n = parseInt(input);

// processamento & saida
for(let i = 1; i < 10000; i++){
    if(i % n === 2) console.log(i);
}

File: 1-Iniciante/ex1075.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int n, i;

    /*entrada*/
    scanf("%d", &n);

    /*processamento & saida*/
    for(i = 2; i < 10001; i++){
        if(i % n == 2){
            printf("%d\n", i);
        }
    }

    return 0;
}